src/migration/ember-addon/steps/use-relative-paths.js

```diff
@@ -12,21 +12,22 @@ function normalizeRelativePath(relativePath) {
 }
 
 function updateFile(oldFile, { filePath, projectName, projectRoot }) {
-  const regex = new RegExp(`\\b${projectName}/(.*/)*(.*)\\b`, 'g');
+  const regex = new RegExp(`(?:'|")(${projectName}/(.*/)*(.*))(?:'|")`, 'g');
```

Comment:
The word boundary `\b` in the beginning led to incorrect matches when the package name is scoped.

By being more specific about the boundaries (i.e. the file path has to be surrounded by single or double quotation marks), we can also avoid false positives, as evidenced in https://github.com/ijlee2/ember-codemod-v1-to-v2/pull/17/commits/0fee1ed584b9220f2ff4d8806b42f9abab1ba862.